src/openrct2/peep/Peep.cpp

```diff
@@ -10754,7 +10754,7 @@ sint32 peep_pathfind_choose_direction(sint16 x, sint16 y, uint8 z, rct_peep * pe
              * In the future these could be used to visualise the
              * pathfinding on the map. */
             uint8          endJunctions         = 0;
-            TileCoordsXYZ endJunctionList[16]  = { 0 };
+            TileCoordsXYZ endJunctionList[16];
```

Comment:
This is not needed.

Reply:
It is with the other change. There's no ctor that accepts '0' as parameter, so...